docs/pages/workflow/customizing.mdx

```diff
@@ -68,8 +64,8 @@ When you're ready to ship your app, you can [build it with EAS Build](/build/int
   cmdCopy="npm i -g eas-cli && eas build -p all"
 />
 
-## Create native modules
+## Create reusable native modules
 
-The [Expo Module API](/modules/overview) enables developers to build modules for Expo and React Native projects using Swift, Kotlin, and TypeScript. We use it for most modules in the [Expo SDK](/versions/latest). Learn more about the [Expo Module API design considerations](/modules/overview).
+As well as being used for adding custom code to single projects, the [Expo Modules API](/modules/overview) enables developers to build reusable modules for Expo and React Native projects using Swift, Kotlin, and TypeScript. These modules are often published to npm; they can also be included as separate packages within a [monorepo](/guides/monorepo). We use the Expo Modules API for most modules in the [Expo SDK](/versions/latest). Learn more about the [Expo Module API design considerations](/modules/overview).
```

Comment:
```suggestion
Apart from adding custom native code to a project, the [Expo Modules API](/modules/overview) enables developers to build reusable modules for Expo and React Native projects using Swift, Kotlin, and TypeScript. These modules are often published to npm. They can also be included as separate packages within a [monorepo](/guides/monorepo). We use the Expo Modules API for most modules in the [Expo SDK](/versions/latest).
```

- I think linking to Expo Modules API overview page once from this is enough
- Also split the sentence instead of using `;`